examples/docker-compose/notebook/README.md

```diff
@@ -0,0 +1,96 @@
+Run a docker-stack notebook container using Docker Compose on a Docker Machine-controlled host.
+
+## Pre-requisites
+
+* [Docker Engine](https://docs.docker.com/engine/) 1.10.0+
+* [Docker Machine](https://docs.docker.com/machine/) 0.6.0+
+* [Docker Compose](https://docs.docker.com/compose/) 1.6.0+
+
+See the [installation instructions](https://docs.docker.com/engine/installation/) for your environment.
+
+## Quickstart
+
+Here's how to build and run a `jupyter/minimal-notebook` container on an existing Docker machine.
```

Comment:
Hmmm. I had to go back and forth between this README and the one a folder up to understand what I'm supposed to do. Can the parent README have a quickstart section? As a new user, I want to come here, see commands to run, and understand what I'm going to get as a result.